packages/ketchup/src/f-components/f-text-field/f-text-field.tsx

```diff
@@ -252,14 +260,25 @@ function setContent(props: FTextFieldProps): HTMLDivElement {
                                     allowNegative: props.allowNegative ?? true,
                                     decimal: props.decimals,
                                     group: props.group,
-                                    integer: props.integers,
+                                    integer: getIntegers(
+                                        props.precision,
+                                        props.decimals
+                                    ),
                                 };
-                                (e.target as HTMLInputElement).value =
-                                    formatValue(
-                                        (e.target as HTMLInputElement).value,
+
+                                const valueFromTarget =
+                                    e.target as HTMLInputElement;
+                                if (valueFromTarget.value) {
+                                    const formattedValue = formatValue(
+                                        valueFromTarget.value,
                                         options,
-                                        true
+                                        true,
+                                        navigator.language
                                     );
+
+                                    valueFromTarget.value = formattedValue;
+                                    props.value = formattedValue;
+                                }
```

Comment:
Why did you change valueFromTarget and props.value?

Reply:
right, I've removed this logic. I've tested and it works